src/qutip_qoc/pulse_optim.py

```diff
@@ -84,6 +84,11 @@ def optimize_pulses(
                 Global steps default to 0 (no global optimization).
                 Can be overridden by specifying in minimizer_kwargs.
 
+            - shorter_pulses : bool, optional
```

Comment:
This is basically what the `__time__` control parameter is used for.
Of course, RL doesn't really take a 'guess' (as with the other parameters), but we could use the same structure to only look for shorter pulses if the `__time__` bounds are specified.